src/space.h

```diff
@@ -100,8 +100,8 @@ class Space {
     json info();
 
     //! Iterable range of all particle positions
-    auto positions() const {
-        return ranges::cpp20::views::transform(p, [](auto &i) -> const Point & { return i.pos; });
+    auto positions() {
```

Comment:
keep both const and non-const version of function?

Reply:
done